_episodes/05-creating-histograms.md

```diff
@@ -70,16 +70,16 @@ The statement
 
 loads up the `pyplot` library, and gives it a shorter name, `plt`.
```

Comment:
@tobyhodges I would like to remove lines 65-71 because they seem out of place (a) it's not the first time in the lesson that have imported `matplotlib`, and b) it's not even imported as `from matplotlib import pyplot as plt` but as `import matplotlib.pyplot as plt`). What do you think?

I'm asking because it's not related to this PR; I would make this change its own commit, of course.

Reply:
I think this is a good suggestion.

Reply:
Done 3a9dbf0064aee846.